test/Test.hs

```diff
@@ -223,6 +224,12 @@ testDELETEQueue queueSID = do
   liftIO . putStrLn . unpack $ "DELETE /Queues/" <> getSID queueSID
   Queue.delete queueSID
 
+{- Recording -}
+testDELETERecording :: RecordingSID -> Twilio ()
+testDELETERecording recordingSID = do
+  liftIO . putStrLn . unpack $ "DELETE /Recordings/" <> getSID recordingSID
+  Recording.delete recordingSID
+
```

Comment:
Nice, so we'll have a function for deleting a recording, although until/unless we generate recordings in the integration test, we won't be able to call this.

Reply:
@markandrus I'll take a look at fixing. BRB.

Reply:
@markandrus Hello. I think I'm going to delete the test and add them as a separate commit, since posting to recordings is not yet implemented. 

I tried to push my branch to this repo but was met with an error:

```
ERROR: Permission to markandrus/twilio-haskell.git denied to StevenXL.
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```

Thank you!